src/bioregistry/data/bioregistry.json

```diff
@@ -40531,7 +40531,8 @@
       "go.chemical",
       "go.chemicals"
     ],
-    "uri_format": "https://biopragmatics.github.io/providers/goche/$1"
+    "uri_format": "http://purl.obolibrary.org/obo/GOCHE_$1",
```

Comment:
it's a mistake to remove this one, since there's no other place to actually resolve these.

Reply:
```suggestion
    "uri_format": "https://biopragmatics.github.io/providers/goche/$1",
```